library/src/main/java/com/airbnb/android/airmapview/WebViewMapFragment.java

```diff
@@ -156,6 +158,15 @@ public void setOnCameraChangeListener(OnCameraChangeListener listener) {
     onCameraChangeListener = listener;
   }
 
+  @Override
+  public void setOnMapInitialisedListener(OnMapInitializedListener listener) {
+    this.onMapInitializedListener = listener;
+    if (loaded) {
+      onMapInitializedListener.onMapInitialized();
```

Comment:
@petzel that's how you guys had it in the original setOnMapLoadedListener. I will remove it